packages/lib-classifier/src/store/feedback/strategies/datavis/graph2drange/feedback-mark.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+function FeedbackMark ({ rule }) {
+  const color = (rule.success) ? 'green' : 'red'
+  const tolerance = parseInt(rule.tolerance, 10)
+  return (
+    <g>
```

Comment:
The classifier is using d3 to draw the subject so I think you’ll need to use d3 to add marks to it. I’m not sure, but I don’t think you can use React components.

Reply:
Ah yes thanks for heads up! I removed for now, will revisit in next PR to show markings.